ui/src/components/yaml/styles.module.less

```diff
@@ -324,17 +324,14 @@
       }
```

Comment:
Class name changed from .yaml_content_diagnosisContent to .yaml_content_diagnosisBody. Ensure all references are updated.